### ISSUE TITLE
Express AutoDocs Workflow Generator

### ISSUE THREAD

**Issue by Pika1998** [Pika1998/express-autodocs#27]
🙋‍♂️Heyya friends, I did some more research on GitHub action and found that people who are not aware of GitHub actions or are new to them, find it difficult to integrate actions and understand the workflow file.

I was thinking if we could create a workflow generator for our action, just to provide a much more comfortable and user friendly approach to integrate our action.

Here I have designed a Figma prototype of the idea in my mind:
## [💡Checkout Workflow Generator Prototype](https://www.figma.com/proto/vq9K1YlHEW5aKOPUygpVag/express-autodocs-workflow-generator?node-id=0%3A1)

# 💖 Would love to have your suggestions & thoughts on this.

**Comment by Pika1998** [comment]
@rahil1304 @rosborne132 @KulkarniSuraj  I would love to have your feedback on this idea :smile:

**Comment by rahil1304** [comment]
Love the idea. I agree with the premise as well. People who are not aware of Github action, either find it difficult to integrate them or don't integrate them due to the fear of messing up their codebase.  @Pika1998

**Comment by rosborne132** [comment]
@Pika1998 Just to make sure I understand what you are suggesting. Is this thought to create a front-end where users fill out a form and it creates a workflow template file? 
I'm not going to lie, I've never used a github action which is what drew me to the project so if you need user testing I would be happy to help. :)

**Comment by Pika1998** [comment]
@rosborne132 that's correct, basically, a front-end to make things easier for users, but I think we should view it as a future scope as of now since there is alot to do in the action itself :smile:

**Comment by KulkarniSuraj** [comment]
That's great idea!
